report/4.Solution/SemanticsDescription.tex

```diff
@@ -0,0 +1,748 @@
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash a_1 \rightarrow_a v_1 \quad env_V,\ sto \vdash a_2 \rightarrow_a v_2}
+	{env_V,\ sto \vdash a_1 < a_2 \rightarrow_b False}
+	\end{math}
+	
+	\texttt{ if } $v_1 >= v_2$
+\end{center}
+
+\textbf{[$GreaterThanTrue_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash a_1 \rightarrow_a v_1 \quad env_V,\ sto \vdash a_2 \rightarrow_a v_2}
+	{env_V,\ sto \vdash a_1 > a_2 \rightarrow_b True}
+	\end{math}
+	
+	\texttt{ if } $v_1 > v_2$
+\end{center}
+
+\textbf{[$GreaterThanFalse_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash a_1 \rightarrow_a v_1 \quad env_V,\ sto \vdash a_2 \rightarrow_a v_2}
+	{env_V,\ sto \vdash a_1 > a_2 \rightarrow_b False}
+	\end{math}
+	
+	\texttt{ if } $v_1 <= v_2$
+\end{center}
+
+\textbf{[$LessThanOrEqualTrue_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash a_1 \rightarrow_a v_1 \quad env_V,\ sto \vdash a_2 \rightarrow_a v_2}
+	{env_V,\ sto \vdash a_1 <= a_2 \rightarrow_b True}
+	\end{math}
+	
+	\texttt{ if } $v_1 <= v_2$
+\end{center}
+
+\textbf{[$LessThanOrEqualFalse_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash a_1 \rightarrow_a v_1 \quad env_V,\ sto \vdash a_2 \rightarrow_a v_2}
+	{env_V,\ sto \vdash a_1 <= a_2 \rightarrow_b False}
+	\end{math}
+	
+	\texttt{ if } $v_1 > v_2$
+\end{center}
+
+\textbf{[$AndTrue_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+		{env_V,\ sto \vdash b_1 \rightarrow_b True \quad env_V, sto \vdash b_2 \rightarrow_b True}
+		{env_V,\ sto \vdash b_1\ and\ b_2 \rightarrow_b True}
+	\end{math}
+\end{center}
+
+\textbf{[$AndFalseBoth_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+		{env_V,\ sto \vdash b_1 \rightarrow_b False \quad env_V,\ sto \vdash b_2 \rightarrow_b False}
+		{env_V,\ sto \vdash b_1\ and\ b_2 \rightarrow_b False}
+	\end{math}
+\end{center}
+
+\textbf{[$AndFalseFirst_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash b_1 \rightarrow_b False \quad env_V,\ sto \vdash b_2 \rightarrow_b True}
+	{env_V,\ sto \vdash b_1\ and\ b_2 \rightarrow_b False}
+	\end{math}
+\end{center}
+
+\textbf{[$AndFalseSecond_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash b_1 \rightarrow_b True \quad env_V,\ sto \vdash b_2 \rightarrow_b False}
+	{env_V,\ sto \vdash b_1\ and\ b_2 \rightarrow_b False}
+	\end{math}
+\end{center}
+
+\textbf{[$Not_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	\cfrac
+	{env_V,\ sto \vdash b \rightarrow_b v_2}
+	{env_V,\ sto \vdash b\ not \rightarrow_b v_1}
+	\end{math}
+	
+	\texttt{ where } $v_1 \rightarrow_b \neg v_2$
+\end{center}
+
+\textbf{[$TFLit_{BSS}$]}\\
+\begin{center}
+	\begin{math}
+	env_V,\ sto \vdash tf \rightarrow_b v
+	\end{math}
+	\texttt{ if } $\mathbb{B}[tf] = v$
+\end{center}
+
+\textbf{\large{Text Expressions}}\\
+Text Expressions returns a value $v$, where $v$ is a sequence of characters from an arbitrary alphabet enclosed in $"$, abbreviated $str$.
```

Comment:
explain that default charset is utf-8-mb4.. :)

Reply:
Fixed